config/webpack/loaders/devSiteCssFinder.js

```diff
@@ -0,0 +1,20 @@
+const findCss = (x) => {
+  /**
+  * The regex picks out lines that don't contain // but do contain some form of quotation (' or "),
+  * and an explicit .css or .scss deceleration
+  */
+  const regex = /^(?!\/\/|\/).*["'](.*\.css|.*\.scss)["']/m;
+  const m = regex.exec(x);
+  let fileName;
+
+  if (m !== null) {
+    m.forEach((match, groupIndex) => {
```

Comment:
what happens if an example has more than 1 css import? do we need to handle that edge case?

Reply:
We will only show the first imported css file. In the majority of cases that will be correct.